doc/source/user_guide/ug_running.rst

```diff
@@ -202,7 +202,10 @@ specifies the compilation environment associated with the machine.  This should
   specifies the grid.  This is a string and for the current CICE driver, gx1, gx3, and tx1 are supported. (default = gx3)
 
 ``--set``,  ``-s`` SET1,SET2,SET3
-  specifies the optional settings for the case.  The settings for ``--suite`` are defined in the suite file.  Multiple settings can be specified by providing a comma deliminated set of values without spaces between settings.  The available settings are in **configurations/scripts/options** and ``cice.setup --help`` will also list them.  These settings files can change either the namelist values or overall case settings (such as the debug flag).
+  specifies the optional settings for the case.  The settings for ``--suite`` are defined in the suite file.  Multiple settings can be specified by providing a comma deliminated set of values without spaces between settings.  The available settings are in **configurations/scripts/options** and ``cice.setup --help`` will also list them.  These settings files can change either the namelist values or overall case settings (such as the debug flag).  For cases and tests (not suites), settings defined in **~/.cice_set** (if it exists) will be included in the --set options.  This behaviour can be overridden with the `--ignore-user-set`` command line option.
```

Comment:
A little suggestion to make reviewing documentation easier: one thing we could do is write only one sentence per line. Here there are multiple sentences on the same line so the diff is long, whereas if we'd stick to one sentence per line it would be easy to see that it's two sentences that are added at the end and the rest of the paragraph is not modified. (Whether to do a PR that changes that for the whole documentation is something we could discuss, but maybe just doing it as we go when we make documentation changes could be a good way forward). Also if everybody is on board we should mention it on the wiki page that talks about Documentation changes.


Also, I think the format of the `~/.cice_set` file should be documented: are options separated by commas, just as on the command line  (and with no trailing comma) ? Reading the code added in `cice.setup` this appears to be the case, but I think it should be mentioned.

Reply:
Thanks @phil-blain, I have updated the documentation and added a sentence about the format of the .cice_set file.  I also agree about the "long line" issue in the documentation and agree that we can move away from it as we go.  I think we have to be a little careful as there may be cases where the line breaks change the formatting, for instance in the indented section highlighted above, but we can also explore that issue moving forward.